docs/installation/configuration-options.md

```diff
@@ -216,6 +216,10 @@ Configures the name of the session key that will be used to hold the original ad
 If `false`, allow the assignment of both roles and permissions to users.
 Set to `true` to restrict user assignments to roles only.
 
+#### disableIpLogging (type: `boolean`, default: `false`)
+
+If `false`, then the users IP is stored in `registration_ip` and `last_login_ip`.
+If set to `true` then the dummy IP `127.0.0.1` is saved.
```

Comment:
I'd just say: *If true registration and last login IPs are not logged into users table, instead a dummy 127.0.0.1 is used*